qulice-maven-plugin/src/test/java/com/qulice/maven/DefaultMavenEnvironmentTest.java

```diff
@@ -128,4 +128,24 @@ public final void passPathsWithWhitespaces()  throws Exception {
             Matchers.notNullValue()
         );
     }
+
+    /**
+     * DefaultMavenEnvironment can produce empty collection when no matches
+     * with checker.
+     * @throws Exception If something wrong happens inside
+     */
+    @Test
```

Comment:
@mkordas yes, sure. I am working TDD style - first test, it fails. Then fix it. Try to revert implementation and run tests - you will see